src/backend/scripts/migrate-all.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
 
 set -Eeuo pipefail
+set -x
```

Comment:
Errrr, actually let's remove this, or move it below the profile writing. This will print all bash commands to the logs, including the keys.